.clang-tidy

```diff
@@ -137,6 +141,7 @@ Checks: [
   '-readability-suspicious-call-argument',
   '-readability-uppercase-literal-suffix',
   '-readability-use-anyofallof',
+  '-readability-math-missing-parentheses',
```

Comment:
Althought this warning seems a bit controversial, I still like it. If there are not tons of occurrences, consider enabling it (up to you).